plugins/uefi-capsule/meson.build

```diff
@@ -1,3 +1,5 @@
+subdir('tests')
+
```

Comment:
shouldn't this only be under `if allow_uefi_capsule`?

Reply:
Good catch, fixed.